code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -900,6 +971,26 @@
 		holder.remove_reagent(/datum/reagent/consumable/capsaicin, 2 * REM * delta_time)
 	..()
 
+/datum/reagent/consumable/italian_coco
+	name = "Italian Hot Chocolate"
+	description = "Made with love! You can just imagine a happy Nonna from the smell."
+	nutriment_factor = 8 * REAGENTS_METABOLISM
+	color = "#57372A"
+	taste_description = "thick creamy chocolate"
+	glass_icon_state = "italiancoco"
+	glass_name = "glass of italian coco"
+	glass_desc = "A spin on a winter favourite, made to please."
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/italian_coco/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	M.adjust_bodytemperature(5 * REM * TEMPERATURE_DAMAGE_COEFFICIENT * delta_time, 0, M.get_body_temp_normal())
+	if(M.getBruteLoss() && DT_PROB(10, delta_time))
+		M.heal_bodypart_damage(1, 0, 0)
```

Comment:
You know that not every single drink needs to have a unique effect right? specially when it's just a random healing one.

If you can't think of anything more intriguing or cant' be bothered coding up something unique then just don't add an effect, don't just fall back on some lazy ass tiny healing

Reply:
Inherits stats from hot coco, it's meant to be a flat upgrade to it.

Reply:
If you want the brute part to be gone then I can budge on this one.